am-lib/src/main/java/uk/gov/hmcts/reform/amlib/repositories/AccessManagementRepository.java

```diff
@@ -41,4 +42,8 @@
     @SqlQuery("select * from access_management where accessor_id=? and resource_id=? LIMIT 1")
     @RegisterConstructorMapper(ExplicitAccessRecord.class)
     ExplicitAccessRecord getExplicitAccess(String accessorId, String resourceId);
+
+    @SqlQuery("select * from default_permissions_for_roles where service_name =? and resource_type =? and resource_name =? and role_name =?")
```

Comment:
Please use named args so that we do not break by ordering args in method signature

Reply:
👍